pkg/agent/openflow/pipeline.go

```diff
@@ -397,105 +398,93 @@ func (c *client) GetTunnelVirtualMAC() net.HardwareAddr {
 	return globalVirtualMAC
 }
 
-func (c *client) Add(flow binding.Flow) error {
+func (c *client) changeAll(flows []binding.Flow, action string) error {
+	if len(flows) == 0 {
+		return nil
+	}
+	var adds, mods, dels []binding.Flow
+	if action == "add" {
+		adds = flows
+	} else if action == "modify" {
+		mods = flows
+	} else if action == "delete" {
+		dels = flows
+	} else {
+		return fmt.Errorf("OF Action not exists: %s", action)
+	}
+
 	startTime := time.Now()
 	defer func() {
 		d := time.Since(startTime)
-		metrics.OVSFlowOpsLatency.WithLabelValues("add").Observe(float64(d.Milliseconds()))
+		metrics.OVSFlowOpsLatency.WithLabelValues(action).Observe(float64(d.Milliseconds()))
```

Comment:
If we have an enum type, we can call its String() func here.

Reply:
Updated.